### DISCUSSION TITLE
doc update?

### DISCUSSION THREAD

**Discussion by tikismoke** [Blackymas/NSPanel_HA_Blueprint#1761] [Q&A] [answered]
Sorry but are this blueprint options stilll available?:
```
    hotwatertemp: sensor.hotwater_temp
    hotwatercharge: switch.charge
    heatingsystemflame: binary_sensor.flamestatus
```

https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/d704de853a029197a2ca8fe9c8466edef0c277d8/docs/howto.md?plain=1#L73-L75

**Comment by edwardtfn** [comment, marked as answer]
No, those keys are not in use anymore. They have been replaced by the sensor values.
I've just updated the docs in `dev` and it will be included in the next release.
